### DISCUSSION TITLE
Seq With Nginx

### DISCUSSION THREAD

**Discussion by xH1SoK4** [datalust/seq-tickets#2062] [Help]
Hi
I'm wondering if someone has nginx reverse proxy for seq

I'm runing linux with Plesk panel and I need a reverse proxy for seq

**Comment by liammclennan** [comment]
Hi @xH1SoK4 

Yes, this is a common setup.

**Reply by xH1SoK4** [reply]
can you provide me the the reverse proxy for nginx please

**Reply by liammclennan** [reply]
I don't have an example at the moment, however it is just a standard nginx reverse proxy configuration as described [here](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/).
